docs/zh/quickstart/openmldb_quickstart.md

```diff
@@ -27,16 +27,26 @@ docker run -it 4pdosc/openmldb:0.4.4 bash
 :::
 
 ````{note}
-本教程将在单个容器中启动 OpenMLDB 单机版或集群版，如果希望从外部访问该容器内的 OpenMLDB 服务端口，启动容器时需要 `-p` 暴露端口。
+本教程将在单个容器中启动 OpenMLDB 单机版或集群版。
+请注意，OpenMLDB 所有 server 默认配置的 endpoint 地址都是 `127.0.0.1`。
+如果希望从本机访问**本机容器内**的 OpenMLDB 服务端口，需要将回环地址改为 `0.0.0.0`，并且在启动容器时需要 `-p` 暴露端口。
```

Comment:
if the host and docker container is in same machine, there is no need to use `-p`, it can also request via `{CONTAINER_IP}:{PORT}` as well. `-p` is needed if host and docker container lives in different physical machine

Reply:
I'll add it.